hoodie-common/src/main/java/com/uber/hoodie/common/table/view/AbstractTableFileSystemView.java

```diff
@@ -216,7 +218,9 @@ private void ensurePartitionLoadedCorrectly(String partition) {
           log.info("Building file system view for partition (" + partitionPathStr + ")");
 
           // Create the path if it does not exist already
-          Path partitionPath = FSUtils.getPartitionPath(metaClient.getBasePath(), partitionPathStr);
```

Comment:
Isn't FSUtils.getPartitionPath working ? Its better to use the method consistently in all places wherever we see
new Path(basePath, relativePartitionPath) to make things work for non-partitioned dataset

Reply:
+1